integrations/nvidia/src/haystack_integrations/utils/nvidia/utils.py

```diff
@@ -3,46 +3,41 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import warnings
-from typing import Any, List, Optional
+from typing import Any, Literal, Optional
 from urllib.parse import urlparse, urlunparse
 
-from .model import MODEL_TABLE, Model
+from .statics import MODEL_TABLE, Model
 
 
-def url_validation(api_url: str, default_api_url: Optional[str], allowed_paths: List[str]) -> str:
+def url_validation(api_url: str) -> str:
     """
     Validate and normalize an API URL.
```

Comment:
Can you add more verbose description of what is the expected format, maybe with a few examples?